tests/queries/0_stateless/01711_decimal_multiplication.sql

```diff
@@ -0,0 +1,4 @@
+SELECT materialize(toDecimal64(4,4)) - materialize(toDecimal32(2,2));
+SELECT toDecimal64(4,4) - materialize(toDecimal32(2,2));
+SELECT materialize(toDecimal64(4,4)) - toDecimal32(2,2);
+SELECT toDecimal64(4,4) - toDecimal32(2,2);
```

Comment:
My 50 cents: the test is called decimal_multiplication, but the test doesn't contain the multiplication tests.